src/se/content.rs

```diff
@@ -39,23 +39,36 @@ macro_rules! write_primitive {
 /// and maps serialized. Unlike `SimpleTypeSerializer` it supports any types in
 /// sequences and serializes them as list of elements, but that has drawbacks.
 /// Sequence of primitives would be serialized without delimiters and it will be
-/// impossible to distinguish between them.
-pub struct ContentSerializer<W: Write> {
+/// impossible to distinguish between them. Even worse, when serializing with
+/// indent, sequence of strings become one big string with additional content
+/// and it would be impossible to distinguish between content of the original
+/// strings and inserted indent characters.
+pub struct ContentSerializer<'i, W: Write> {
     pub writer: W,
     /// Defines which XML characters need to be escaped in text content
     pub level: QuoteLevel,
+    /// Current indentation level
+    pub(super) indent: Indent<'i>,
```

Comment:
The documentation should probably make the distinction between `write_indent: false` and `indent: Indent::None` more clear

Reply:
Explained:
> Current indentation level. Note, that `Indent::None` means that there is no indentation at all, but `write_indent == false` means only, that indent writing is disabled in this instantiation of `ContentSerializer`, but child serializers should have access to the actual state of indentation.